rest/handler.go

```diff
@@ -551,14 +551,14 @@ func (h *handler) disableResponseCompression() {
 // If status is nonzero, the header will be written with that status.
 func (h *handler) writeJSONStatus(status int, value interface{}) {
 	if !h.requestAccepts("application/json") {
-		base.Warn("Client won't accept JSON, only %s", h.rq.Header.Get("Accept"))
+		base.WarnR("Client won't accept JSON, only %s", base.MD(h.rq.Header.Get("Accept")))
```

Comment:
Not convinced this needs redacting, but feel free to persuade me :)

Reply:
> Not convinced this needs redacting, but feel free to persuade me :)

Ok I think I may have actually not fully understood the original goal here.  I was thinking that the goal was to *annotate* everything in one of the three categories:

- `base.UD()`
- `base.MD()`
- `base.SD()`

but I think it's more about identifying data that needs to be redacted, and if it can contain any sensitive info, then further annotate it with the types above.  

I'll remove redaction here and a bunch of other places that don't contain data that requires redaction.